tests/index_test.js

```diff
@@ -118,6 +119,44 @@ describe('css', () => {
             done();
         });
     });
+
+    it('adds extra class names from descendants', () => {
+        const sheet = StyleSheet.create({
+            base: {
+                '>>child': {
+                    color: 'red',
+                }
+            },
+
+            blue: {
+                color: 'blue',
+            },
+        });
+
+        assert.equal(
+            css(sheet.blue, sheet.base.child),
```

Comment:
What happens if you have `css(sheet.base.child, sheet.blue)`? I'm guessing behaviour is reasonable, but would be good to encode in a test.